src/Tizen.Security/Tizen.Security/Privilege.cs

```diff
@@ -51,11 +51,12 @@ internal static string ToPackageTypeString(PackageType type)
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
         /// <remarks>If there's no matching privilege, then it returns last token of the given privilege.</remarks>
+		/// <remarks>Since Tizen 6.0, if there's no matching privilege then it returns ArgumentException. </remarks>
```

Comment:
Please write it in a single `<remarks>` tag, because some of our API analysis tools may ignore your changes. It applies also to your other documentation comments.

Reply:
Done.

Reply:
You have merged tags for `GetDescription(string apiVersion, string privilege, PackageType packageType)` and `GetDescription(string apiVersion, string privilege)`, but not for `GetDisplayName(string apiVersion, string privilege)` and `GetDisplayName(string apiVersion, string privilege, PackageType packageType)`. Please update them too.

Reply:
Merged.